### ISSUE TITLE
magento 2.2.5 + latest es stops compilation

### ISSUE THREAD

**Issue by lingwooc** [Smile-SA/elasticsuite#987]
You are probably aware of this already but 2.2.5 breaks es.

### Preconditions

Magento Version : 2.2.5 CE

ElasticSuite Version : 2.5.12

Environment : Production

### Steps to reproduce

1. Install magento 2.2.5 and elastic suite 
2. Compile

### Expected result

1. Compiles

### Actual result

1.
```
Declaration of Smile\ElasticsuiteCore\Model\ResourceModel\Search\Request\RelevanceConfig::saveConfig($path, $value, $scope, $scopeCode = 'default') 
must be compatible with Magento\Framework\App\Config\ConfigResource\ConfigInterface::saveConfig($path, $value, $scope = Magento\Framework\App\Config\ScopeConfigInterface::SCOPE_TYPE_DEFAULT, $scopeId = 0) 
in /home/jenkins/workspace/MagentoStaging/magento2/shop/vendor/smile/elasticsuite/src/module-elasticsuite-core/Model/ResourceModel/Search/Request/RelevanceConfig.php on line 99
```

Gotta love breaking changes in minor versions!

**Comment by afoucret** [comment]
PR #990 should fix the bug and will be part of the next bugfix release (2.5.13).

Thx for reporting.

**Comment by mklooss** [comment]
perfect, running in to the same issue :)

**Comment by VincentMarmiesse** [comment]
Hi @afoucret,
Do you know the release date of this fix?

**Comment by afoucret** [comment]
PR have been merged into 2.5.x branch.

The release 2.5.13 will be out when #991 will be fixed.

**Comment by afoucret** [comment]
It seems there is huge expectations on this .

Just tweeted the release : https://twitter.com/afoucret/status/1012291355056443392
